core/src/main/resources/lib/form/hetero-list.jelly

```diff
@@ -65,6 +65,12 @@ THE SOFTWARE.
       If true, insert new addition by default to their 'desired' location, which
       is the order induced by the descriptors.
     </st:attribute>
+    <st:attribute name="enableTopButton">
+      true if a new Add button, for inserting a new item, should be displayed above the existing items
+      Display of top button depends also on number of items. If there is no item, only one button is displayed. When at least one item is present, there are two
+      buttons displayed (only when enableTopButton is true). One above and one below.
+      Top button adds item on top  Bottom button adds item on the bottom. When honorOrder is set the item is inserted at its 'desired' location.
```

Comment:
```suggestion
      Top button adds item on top  Bottom button adds item on the bottom. When honorOrder is set the item is inserted at its 'desired' location.
      @since TODO
```